x-pack/plugins/observability_ai_assistant/public/components/chat/chat_prompt_editor.tsx

```diff
@@ -125,20 +131,24 @@ export function ChatPromptEditor({
               arguments: currentPayload,
             },
           },
-        });
+        };
+        onSubmit(message);
```

Comment:
Is there some risk to not longer awaiting this call?

Reply:
`onSubmit` was not returning a promise so the `await` wasn't doing anything here